src/poliastro/plotting.py

```diff
@@ -52,11 +52,29 @@ def __init__(self, ax=None, num_points=100):
         num_points : int, optional
             Number of points to use in plots, default to 100.
 
+        bgcolor : tuple or string, optional
+            Background color of the plot, default to (1,1,1)
+
+        linewidth : int, optional
+            Width of the dashed lines showing the orbit path, default to 1.5
+
+        markersize : int, optional
+            Size of the point representing individual orbit bodies, default to 6
+
+        audivision : int, optional
```

Comment:
Good description! I'm just not comfortable with the name of the parameter, as stated above. Perhaps `scale_factor` or something?

Reply:
Hello! Sorry for the late resolve; I am very, very bad at time management, but just remembered this pending commit.

Are you talking about `markersize`, or `audivision`?

Reply:
Hi! Don't worry, you caught me by surprise :) I was talking about `audivision`. Anyway, this branch now has conflicts so perhaps it would be better to start over. Would you accept doing so?